src/include/migraphx/onnx.hpp

```diff
@@ -11,6 +11,7 @@ inline namespace MIGRAPHX_INLINE_NS {
 struct onnx_options
 {
     unsigned int batch_size = 1;
+    std::unordered_map<std::string, std::vector<std::size_t>> map_input_dims;
```

Comment:
Why are we mapping to the inputs and not filling in the dim_params?

Reply:
It seems that exported onnx files have a dim_param associated with the dim that is dynamic, but the name seems to be randomly generated (ex. "unk__271"). If there is a case where there is no dim_param in the onnx file but the shape is dynamic, we still need a way to handle that.

Reply:
This should be initialized with `std::unordered_map<std::string, std::vector<std::size_t>> map_input_dims{};` or `std::unordered_map<std::string, std::vector<std::size_t>> map_input_dims = {};`